### PR TITLE
Document \u{x...} unicode escapes

### PR DESCRIPTION
**Pull request by bobhy**
…ed review.

### CODE REVIEW

File: de/book/working_with_strings.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Nushell unterstützt aktuell folgende Escape Zeichen:
 - `\r` - Zeilenumschalt
 - `\n` - Neue Zeile (line feed)
 - `\t` - Tabulator
-- `\uXXXX` - Ein Unicode Zeichen (Tausche XXXX mit der Nummer des Unicode Zeichens)
+- `\u{X...}` - Ein Unicode Zeichen, wobei X... 1-6 hexadezimale Ziffern sind (0-9, A-F)
```
Comment (Hofer-Julian) [inline de/book/working_with_strings.md:52]:
```suggestion
- `\u{X...}` - Ein Unicode Zeichen, wobei der Platzhalter `X...` für ein bis sechs hexadezimale Ziffern steht (0-9, A-F)
```

**Comment by sholderbach** [comment]
@hustcer @WindSoilder If you have a minute a check of the translation would be appreciated :)

**Review by WindSoilder — COMMENTED** [review]
Just leave a comment about chinese description

File: zh-CN/book/working_with_strings.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Nushell 目前支持以下转义字符：
 - `\r` - 回车符
 - `\n` - 换行符 (line feed)
 - `\t` - 制表符
-- `\uXXXX` - Unicode 字符 (用 Unicode 字符的编号替换 XXXX)
+- `\u{X...}` - Unicode字符（将X...替换为1到6个十六进制数字）
```
[the remark this answers is not in the document]
Comment (sholderbach) [reply zh-CN/book/working_with_strings.md]: 谢谢

**Pull request merged**